manifests/osfamily/debian.pp

```diff
@@ -84,8 +84,10 @@
     $keyname = 'GPG-KEY-puppet'
     $gpg_path = "/etc/pki/deb-gpg/${keyname}"
 
-    file { ['/etc/pki', '/etc/pki/deb-gpg']:
-      ensure => directory,
+    if getvar('::puppet_agent::manage_pki_dir') == true {
+      file { ['/etc/pki', '/etc/pki/deb-gpg']:
```

Comment:
Is this something you manage elsewhere? This is something I really wish we'd fix in Puppet.

Reply:
Yes, some of our profiles manage this resource.  And in general, since this module is about managing the puppet agent, rather than gpg keys, I would avoid managing this particular directory within this module.